docker/startCodex.sh

```diff
@@ -1,7 +1,15 @@
-echo "Starting Codex..."
+NAME=""
+if [ -n "$CODEX_NODENAME" ]; then
+  NAME=" '$CODEX_NODENAME'"
+fi
+echo "Starting Codex node$NAME"
 
 args=""
 
+## local ip as NAT?
+do_nat=$(hostname -i)
```

Comment:
I don't think this is cross-platform compatible. On my mac,
```shell
❯ which hostname
/bin/hostname

❯ hostname -i
hostname: illegal option -- i
usage: hostname [-f] [-s | -d] [name-of-host]

❯ hostname
Es-MacBook-Pro.local
```

Reply:
I believe that immediately, but I don't think it's a problem:
This runs in a docker, it's always on ubuntu/lunar.
In the future, this nat-workaround should be replaced by proper nat-behavior inside Codex. (It might not be applicable inside docker or the dist-test environment though... 🤔)

Reply:
Good point. 👍